modules/installation-ibm-cloud-config-yaml.adoc

```diff
@@ -203,6 +207,101 @@ For production {product-title} clusters on which you want to perform installatio
 ====
 endif::vpc[]
 
+ifdef::private[]
+[source,yaml]
+----
+apiVersion: v1
+baseDomain: example.com <1>
+controlPlane: <2> <3>
+  hyperthreading: Enabled <4>
+  name: master
+  platform:
+    ibm-cloud: {}
+  replicas: 3
+compute: <2> <3>
+- hyperthreading: Enabled <4>
+  name: worker
+  platform:
+    ibmcloud: {}
+  replicas: 3
+metadata:
+  name: test-cluster <1>
+networking:
+  clusterNetwork:
+  - cidr: 10.128.0.0/14 <5>
+    hostPrefix: 23
+  machineNetwork:
+  - cidr: 10.0.0.0/16
+ifndef::openshift-origin[]
+  networkType: OpenShiftSDN
+endif::openshift-origin[]
+ifdef::openshift-origin[]
+  networkType: OVNKubernetes
+endif::openshift-origin[]
+  serviceNetwork:
+  - 172.30.0.0/16
+platform:
+  ibmcloud:
+    region: eu-gb <1>
+    resourceGroupName: eu-gb-example-network-rg <6>
+    vpcName: eu-gb-example-network-1 <7>
+    controlPlaneSubnets: <8>
+      - eu-gb-example-network-1-cp-eu-gb-1
+      - eu-gb-example-network-1-cp-eu-gb-2
+      - eu-gb-example-network-1-cp-eu-gb-3
+    computeSubnets: <9>
+      - eu-gb-example-network-1-compute-eu-gb-1
+      - eu-gb-example-network-1-compute-eu-gb-2
+      - eu-gb-example-network-1-compute-eu-gb-3
+credentialsMode: Manual
+publish: Internal <10>
+pullSecret: '{"auths": ...}' <1>
+ifndef::openshift-origin[]
+fips: false <11>
+sshKey: ssh-ed25519 AAAA... <12>
+endif::openshift-origin[]
+ifdef::openshift-origin[]
+sshKey: ssh-ed25519 AAAA... <11>
+endif::openshift-origin[]
+capabilities:
+  baselineCapabilitySet: None
+  additionalEnabledCapabilities:
+  - openshift-samples
+----
+<1> Required. The installation program prompts you for this value.
```

Comment:
for private cluster use the dns zone, not prompt by installation program. 
$ ic dns instances
Name                             ID                                     Location   State    Service Name   
preserved-openshift-qe-private   54cc61f8-28bc-4f9a-a815-e07dfe705fed   global     active   dns-svcs   
$ ic dns zones 
Listing zones for service instance 'preserved-openshift-qe-private' ...
ID                                     Name                                           Status   
3c7af30d-cc2c-4abc-94e1-3bcb36e01a9b   **private-ibmcloud.qe.devcluster.openshift.com** PENDING_NETWORK_ADD

Reply:
This is an excellent catch. I had completely overlooked the fact that you cannot use the installation program to create the the `install-config.yaml` file. You must, in fact, create it manually. I have gone ahead and added the correct procedure for doing that.